resources/assets/lib/core-legacy/store-checkout.coffee

```diff
@@ -33,7 +33,8 @@ export class StoreCheckout
       showLoadingOverlay.flush()
 
       init[provider]?.then ->
-        $.post(route('store.checkout.store'), { provider, orderId })
+        hide_from_activity = document.querySelector('.js-store-checkout-option')?.checked
```

Comment:
the class name ("store checkout option") and what it does doesn't match ("hide from activity")